docs/core/tools/dotnet-msbuild.md

```diff
@@ -40,4 +40,8 @@ Build a project and its dependencies using Release configuration:
 
 Run the publish target and publish for the `osx.10.11-x64` RID:
 
-`dotnet msbuild /t:Publish /p:RuntimeIdentifiers=osx.10.11-x64`
\ No newline at end of file
+`dotnet msbuild /t:Publish /p:RuntimeIdentifiers=osx.10.11-x64`
+
+See the whole project with all targets included by the SDK:
+
+`dotnet msbuild /pp`
```

Comment:
Similar to @rainersigwald feed back above, we should probably just include an output filename here.